flight/PiOS/Common/Libraries/ChibiOS/os/common/startup/ARMCMx/compilers/GCC/crt0_v7m.S

```diff
@@ -52,6 +52,15 @@
 /* Module pre-compile time settings.                                         */
 /*===========================================================================*/
 
+/* dRonin: This if fixed upstream apparently, remove on upgrade to 18.x. */
```

Comment:
Can never be fixed because we will always have old bootloaders around.

Reply:
I mean the CRT0_FORCE_MSP_INIT thing. It's not set at all in this ChibiOS release. It's been set to TRUE by default in 18.x.